client/src/pages/recording-view/recording-view.tsx

```diff
@@ -59,6 +62,9 @@ export function DisplaySpectrogram() {
         <Stage width={50} height={spectrogramHeight} className="mr-1">
           <RulerSide currentRowAtTop={currentFFT} />
         </Stage>
+        <Stage width={MINIMAP_FFT_SIZE + 5} height={spectrogramHeight}>
```

Comment:
What does the +5 do?

Reply:
DOnt know just copied what ever was the way to calculate this in the previous page the way this ccalculates I think it gets the size of the fft plus adds 5 pixels for the small which squares that you see on the side for downloaded tiles.